src/main/java/de/berlin/lostberlin/ApplicationConfig.java

```diff
@@ -0,0 +1,10 @@
+package de.berlin.lostberlin;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@Configuration
+@EnableJpaAuditing
+public class ApplicationConfig {
+
```

Comment:
Here it is - JPA is initialized here when you run the app. And it is not expected to be loaded when you mock it in the test.

Reply:
Ah ok!